pokedex_async_redux/lib/features/pokemon_overview_page.dart

```diff
@@ -10,25 +11,50 @@ class PokemonOverviewPage extends StatelessWidget {
     Key? key,
   }) : super(key: key);
 
-  final List<Pokemon> pokemons;
+  final Async<List<Pokemon>> pokemons;
+
+  void _showErrorMessageSnackbar(BuildContext context, String message) {
+    WidgetsBinding.instance.addPostFrameCallback(
```

Comment:
see cleo's pr for reference on how she segregated the snackbar implementation with the postFrameCallback 🙂